sentinel/src/db.js

```diff
@@ -50,6 +55,13 @@ if (UNIT_TEST_ENV) {
   module.exports.sentinel = new PouchDB(`${couchUrl}-sentinel`, {
     fetch: fetchFn,
   });
+
+  module.exports.allDbs = () => new Promise((resolve, reject) => {
+    request({ url: `${module.exports.serverUrl}/_all_dbs`, json: true }, (err, response, body) => {
+      return err ? reject(err) : resolve(body);
+    });
+  });
+  module.exports.metaDb = db => new PouchDB(`${module.exports.serverUrl}/${db}`);
```

Comment:
I'm not sure about the naming of this function. [In API](https://github.com/medic/medic/blob/master/api/src/db.js#L61) we call this `get` so it might be worth being consistent, unless you can think of a better name?

Reply:
Changed to `get`.